tests/test_processor_remote_tiles_westport/instruction.json

```diff
@@ -18,10 +18,11 @@
 			"benchmark_dem": "tests/test_processor_remote_tiles_westport/data/benchmark_dem.nc"
 		},
 		"apis": {
-			"open_topography": "True"
+			"open_topography": true
 		},
 		"general": {
-			"set_dem_shoreline": "True"
+			"set_dem_shoreline": true,
+			"drop_offshore_lidar": false
```

Comment:
did you test with "droping the offshore lidar" for all your test cases?

Reply:
Three tests have is set to False, and one test has it set to True. That test is the 'test_processor_remote_all_westport'